src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,6 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+CA1839 | Performance | Info | UseEnvironmentMembers, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1839)
+CA1840 | Performance | Info | UseEnvironmentMembers, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1840)
```

Comment:
Shouldn't CA1837 be reused based on https://github.com/dotnet/runtime/issues/43257#issuecomment-713061245?

Reply:
I don't think using the same diagnostic ID is a good idea here.  The target APIs were released in different versions of the libraries, and I think someone who's multitargeting will want to be able to suppress diagnostics for the APIs that don't exist in all of the platforms they're targeting while still getting diagnostics for those that do exist everywhere.  I'll need to rewatch the video to understand why we said that.